pkg/dashboards/types.go

```diff
@@ -166,7 +166,7 @@ type DashboardLineWidgetConfigurationInput struct {
 // DashboardMarkdownWidgetConfigurationInput - Configuration for visualization type 'viz.markdown'
 type DashboardMarkdownWidgetConfigurationInput struct {
 	// Markdown content of the widget
-	Text string `json:"text"`
+	Text string `json:"text,omitempty"`
```

Comment:
Should go back and test if this is required now that `Markdown` below would be `nil`